.github/s2n_doxygen.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+set -eu
+
+export DOXYGEN_VERSION="doxygen-1.9.3"
+curl -L "https://www.doxygen.nl/files/$DOXYGEN_VERSION.linux.bin.tar.gz" -o "$DOXYGEN_VERSION.tar.gz"
+tar -xvf "$DOXYGEN_VERSION.tar.gz"
+
+# Pull in git tags
+git fetch origin --tags
+
+# Add a version to the Doxygen documentation
+# For example: v1.3.13-3b413f18
+DOC_VERSION="$(git tag | sort | tail -n 1)-$(git rev-parse --short=8 HEAD)"
```

Comment:
Sort isn't doing the right thing here (at least on mac), and the numerical flag doesn't help...

Reply:
Thanks for catching this! Pushed a version that properly sorts the semver.